src/controller/brainState.ts

```diff
@@ -0,0 +1,23 @@
+export const UseCases = { News: 1, Entertainment: 2, DailyRoutine: 3 };
+Object.freeze(UseCases);
+let chatId = 857438551;
+// eslint-disable-next-line import/no-mutable-exports
+export let interruptedUseCase = false;
+// eslint-disable-next-line import/no-mutable-exports
+export let activeUseCase;
+
+export function interruptUseCase() {
+  interruptedUseCase = true;
+}
+export function reset() {
+  interruptedUseCase = false;
+}
+export function activateUseCase(selectedUsecase) {
+  activeUseCase = selectedUsecase;
+}
+export function setChatId(newChatId) {
+  chatId = newChatId;
+}
+export function getChatId() {
+  return chatId;
+}
```

Comment:
In einer Klasse und nicht als einzelne Variablen kann das eleganter sein

Reply:
fixed it

Reply:
Ich hatte eigentlich an alles (also alle Zeilen inklusive Funktionen) in einer Klasse gedacht, die dann z.B. im MessageHandler instanziiert wird gedacht. Sorry, dass ich mich da unklar ausgedrückt habe.